doc/source/msg.rst

```diff
@@ -104,27 +187,44 @@ of immediate motion
 trigger
 +++++++
 
+.. automethod:: bluesky.run_engine.RunEngine._trigger
```

Comment:
I added these but the docs don't look so nice with them. It would be nice to take the docstring just from these functions and paste them here.

Reply:
Tough call. Reuse is good, especially because it help us avoid accidental inconsistencies, but readability is also good.